development/wasm-filter-plugins.md

```diff
@@ -129,6 +129,36 @@ For example, one of the examples of [Rust WASM filter](https://github.com/fluent
 [0] dummy.local: [1666270590.271107000, {"lang"=>"Rust", "message"=>"dummy", "original"=>"{"message":"dummy"}", "tag"=>"dummy.local", "time"=>"2022-10-20T12:56:30.271107000 +0000"}]
 ```
 
+Another example of a Rust WASM filter is called [flb_filter_iis](https://github.com/kenriortega/flb_filter_iis), this filter takes the IIS w3c logs (with some custom modifications) and transforms the raw string into a fluentbit json standard.
```

Comment:
Split into two lines.
```suggestion
Another example of a Rust WASM filter is the [flb_filter_iis](https://github.com/kenriortega/flb_filter_iis) filter.
This filter takes the IIS w3c logs (with some custom modifications) and transforms the raw string into a fluentbit json standard.
```

Clarify all these acronyms (e.g. IIS) with links out to explain them.
Ideally add some sample input to demonstrate it too.